src/main/kotlin/ink/abb/pogo/scraper/Bot.kt

```diff
@@ -85,6 +86,7 @@ class Bot(val api: PokemonGo, val settings: Settings) {
         fixedRateTimer("BotLoop", false, 0, 5000, action = {
             thread(block = {
                 task(keepalive)
+                task(evolve)
```

Comment:
I like this moved down after task(catch). Otherwise those Pidgeys get released before we can evolve them for the XP.